### PR TITLE
docs(gui): update docs for Kong Manager

### PR DESCRIPTION
**Pull request by raoxiaoyan**
### Testing instructions

Preview link: 
https://deploy-preview-7976--kongdocs.netlify.app/
### Checklist 

- [x] Review label added 
- [ ] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

KM-516

### CODE REVIEW

File: app/_src/gateway/kong-manager/enable.md [hunk 1]
```diff
@@ -6,6 +6,18 @@ If you're running {{site.base_gateway}} with a database (either in traditional
 or hybrid mode), you can enable {{site.base_gateway}}'s graphical user interface
 (GUI), Kong Manager.
 
+{:.note}
+> **Note**: To configure Kong Manager to be accessible from multiple domains, you can list the domains as comma-separated values and assign them to the `admin_gui_url` parameter in your Kong configuration. Here's an example:
```
Comment (lena-larionova) [inline app/_src/gateway/kong-manager/enable.md:10]:
```suggestion
> **Note**: To configure Kong Manager to be accessible from multiple domains, you can list the domains as comma-separated values in the `admin_gui_url` parameter in your Kong configuration. For example:
```

File: app/_src/gateway/kong-manager/enable.md [hunk 2]
```diff
@@ -6,6 +6,18 @@ If you're running {{site.base_gateway}} with a database (either in traditional
...
+  ```
+  admin_gui_url = http://localhost:8002, http://127.0.0.1:8002
+  ```
+> If you are set the `admin_gui_path` in your Kong configuration as well.
```
Comment (lena-larionova) [inline app/_src/gateway/kong-manager/enable.md:14]: "If you are" what? Looks like this is missing a couple of words.

**Review by lena-larionova — COMMENTED** [review]
Is this currently possible, or an upcoming feature?

**Comment by raoxiaoyan** [comment]
@lena-larionova Please wait for this PR to be merged. https://github.com/Kong/kong-ee/pull/10260

File: app/_src/gateway/kong-manager/enable.md [hunk 3]
```diff
@@ -6,6 +6,18 @@ If you're running {{site.base_gateway}} with a database (either in traditional
 or hybrid mode), you can enable {{site.base_gateway}}'s graphical user interface
 (GUI), Kong Manager.
 
+{:.note}
```
Comment (lena-larionova) [inline app/_src/gateway/kong-manager/enable.md:9]:
```suggestion
{% if_version gte:3.9.x %}

{:.note}
```

File: app/_src/gateway/kong-manager/enable.md [hunk 4]
```diff
@@ -6,6 +6,18 @@ If you're running {{site.base_gateway}} with a database (either in traditional
...
+  ```
+  admin_gui_url = http://localhost:8002, http://127.0.0.1:8002
+  ```
+> If set the `admin_gui_path` in your Kong configuration as well.
+  ```
+  admin_gui_url = http://localhost:8002/manager, http://127.0.0.1:8002/manager
+  admin_gui_path = /manager
+  ```
+> Make sure that each domain has proper DNS records and that the Kong instance is accessible from all specified domains.
+
```
Comment (lena-larionova) [inline app/_src/gateway/kong-manager/enable.md:20]:
```suggestion
{% endif_version %}
```

File: app/_src/gateway/kong-manager/enable.md [hunk 5]
```diff
@@ -6,6 +6,18 @@ If you're running {{site.base_gateway}} with a database (either in traditional
 or hybrid mode), you can enable {{site.base_gateway}}'s graphical user interface
 (GUI), Kong Manager.
 
+{:.note}
+> **Note**: To configure Kong Manager to be accessible from multiple domains, you can list the domains as comma-separated values in the `admin_gui_url` parameter in your Kong configuration. For example:
+  ```
+  admin_gui_url = http://localhost:8002, http://127.0.0.1:8002
+  ```
+> If set the `admin_gui_path` in your Kong configuration as well.
```
Comment (lena-larionova) [inline app/_src/gateway/kong-manager/enable.md:14]:
```suggestion
> If set, update the `admin_gui_path` in your Kong configuration as well:
```

**Review by lena-larionova — APPROVED** [review]

**Comment by lena-larionova** [comment]
Merging, this will show up under the "unreleased" tag.

**Pull request merged**